ccd-definition/CaseEventToFields/uploadDocumentsAfterGatekeeping.json

```diff
@@ -0,0 +1,135 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "uploadDocumentsAfterGatekeeping",
+    "CaseFieldID": "uploadDocuments_paragraph_1",
+    "PageFieldDisplayOrder": 1,
+    "DisplayContext": "READONLY",
+    "PageID": 1,
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "CallBackURLMidEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/enter-social-work-other/mid-event"
```

Comment:
is this callback needed? @akomisarek .. it's in upload documents but not upload documents after submission

Reply:
I think so, I have added to the submission.